### PR TITLE
Dev 695 webapp must request past events to microservice

### PR DESCRIPTION
**Pull request by clarkjoao**

### CODE REVIEW

**Review by vhcsilva — APPROVED** [review]

**Review by MarcusviniciusLsantos — APPROVED** [review]

File: scripts/deploy-v2.js [hunk 1]
```diff
@@ -92,7 +92,7 @@ const networks = {
 
 
 async function main() {
-
+  console.log({ host: process.env.NEXT_DB_DATABASE, ...process.env })
```
Comment (moshmage) [inline scripts/deploy-v2.js:95]: use `console.table`

Comment (clarkjoao) [reply scripts/deploy-v2.js:95]: I will remove, it was only to try see the envs in Jenkins

File: components/price-conversor-modal.tsx [hunk 2]
```diff
@@ -53,9 +53,13 @@ export default function PriceConversorModal({
 
   useEffect(()=>{
     const currencyList = settings?.currency?.conversionList || defaultValue;
-    const opt = currencyList.map(currency=>({value: currency?.value, label: currency?.label}))
-    setOptions(opt)
-    handlerChange(opt[0])
+    console.log(currencyList)
```
Comment (moshmage) [inline components/price-conversor-modal.tsx:56]: lingering console-log

**Review by moshmage — APPROVED** [review]
lgtm

**Comment by moshmage** [comment]
@clarkjoao I just noticed that since most of the events will run at the same time, we might get conflicting states (pex, creating and immedeatly canceling a bounty would trigger the action to go hay wire) -- we need to offset the timers by 1 at least

(every 30mins, 31mins, 32mins, etc)

**Pull request merged**